README.md

```diff
@@ -25,90 +20,55 @@ Here are the method signatures as definied in the `StringPreference` class.
 
 ```java
 public class StringPreference {
-    public String get() { ... }
-    public boolean isSet() { ... }
-    public void set(String value) { ... }
-    public void delete() { ... }
+    public String get() {  }
+    public boolean isSet() {  }
+    public void set(String value) {  }
+    public void delete() {  }
 }
 ```
 
-Wrappers for frequently used classes can be found and added in the [wiki][typedpreferences-wiki]. Take a look!
+## Source
 
+Inspired by https://github.com/johnjohndoe/TypedPreferences - version 1.0
 
-## How to get the thing running?
-
-Add the library as a dependency to your `build.gradle` in case you use Gradle:
 
+## Dependency
+1. For using apprater module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
 ```groovy
-dependencies {
-    compile "info.metadude.android:typed-preferences:{$version}"
+    dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.har'])
+    implementation project(path: ':library')
+    testImplementation 'junit:junit:4.13'
+    ohosTestImplementation 'com.huawei.ohos.testkit:runner:1.0.0.100'
 }
 ```
-Or if you use Maven you should be looking for this:
+2. For using apprater in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
+```groovy
+	dependencies {
+		implementation fileTree(dir: 'libs', include: ['*.har'])
+		testImplementation 'junit:junit:4.13'
+	}
+```
+## Usage
 
+#### Include following code in your layout:
 ```xml
-<dependency>
-    <groupId>info.metadude.android</groupId>
-    <artifactId>typed-preferences</artifactId>
-    <version>{$version}</version>
-</dependency>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is generic code not relevant to the typedPreferences library. Add the correct code snippet.